source/shared/net/concepts/concepts-transactions.rst

```diff
@@ -0,0 +1,45 @@
+.. _transactions-overview:
+
+=====================
+Transactions overview
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+
+A transaction on the Concordium blockchain is an operation which applies some change to the chain. All transactions are recorded on the chain and once recorded, they are immutable. A transaction always has one sender :ref:`account<glossary-account>` and is signed using the keys of this account.
+
+The most basic transaction is the GTU transfer that is used to send GTU from one account to another. However, there are several transaction types on the Concordium blockchain:
+
+- Send GTU
+- Send GTU with a schedule
+- Shield GTU
+- Unshield GTU
+- Make shielded transfer
+- Add baker
+- Remove baker
+- Update baker stake
+- Update restake earnings
+- Update baker keys
+- Update account credentials
```

Comment:
Are smart contract transactions deliberately omitted?

Reply:
I think, we'll leave smart contracts out for now because the smart contracts docs are currently separate from the rest. It might make sense to add them later, though.